code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -744,6 +744,12 @@
 	if(speech_pref)
 		H.mind.speech_span = "wingdings"
 
+/datum/species/grey/handle_reagents(mob/living/carbon/human/H, datum/reagent/R)
+	if(R.id == "sacid")
+		H.reagents.del_reagent(R.id)
+		return 0
+	return 1
```

Comment:
`return ..()`

Reply:
The doccumentation states
// Return 1 if it should do normal processing too
// Return 0 if it shouldn't deplete and do its normal effect
Should this be updated too?